master/internal/api_experiment.go

```diff
@@ -1587,6 +1587,9 @@ func (a *apiServer) fetchTrialSample(trialID int32, metricName string, metricTyp
 	var zeroTime time.Time
 	var err error
 	var trial apiv1.TrialsSampleResponse_Trial
+	var metricMeasurements db.MetricMeasurements
+	xAxisLabelMetrics := []string{"epoch"}
```

Comment:
could this maybe go outside the function and in `ALL_CAPS` to indicate that it is a constant?